### PR TITLE
[OneExplorer] Relieve function complexity

### PR DESCRIPTION
**Pull request by dayo09**
This commit is to relieve function complextiy in test.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

**Comment by dayo09** [comment]
cc(Cyclomatic complexity) violation number decreased from 112 to 65

**Review by jyoungyun — APPROVED** [review]
LGTM

File: src/Tests/OneExplorer/ConfigObject.test.ts [hunk 1]
```diff
@@ -46,16 +46,15 @@ suite("OneExplorer", function () {
       });
 
       test("NEG: Create config of a file without any valid content", function () {
-        const configName = "model.cfg";
```
Comment (shs-park) [inline src/Tests/OneExplorer/ConfigObject.test.ts:49]: I think using const seems to be better for code readability, but it seems to increase CC violations(!!). 
I don't understand why using simple constants/variables is related to cyclomatic complexity score.
😢

**Review by shs-park — APPROVED** [review]
The code seems to good to me, but I left a question.

File: src/Tests/OneExplorer/ConfigObject.test.ts (hunk 1 above)
Comment (dayo09) [reply src/Tests/OneExplorer/ConfigObject.test.ts:49]: I totally agree... I just followed the guide anyways. 
In terms of readability, I first thought that using const is better but some people told me that writing down the file name is better... 🤔

**Pull request merged**